src/home/homework.c

```diff
@@ -1853,7 +1853,7 @@ static int user_session_freezer(uid_t uid, bool freeze_now, UnitFreezer *ret) {
 
 static int home_lock(UserRecord *h) {
         _cleanup_(home_setup_done) HomeSetup setup = HOME_SETUP_INIT;
-        _cleanup_(unit_freezer_done_thaw) UnitFreezer freezer = {};
+        _cleanup_(unit_freezer_thaw_done) UnitFreezer freezer = {};
```

Comment:
I put it in this order because throughout the codebase the pattern seems to be `<type>_<done|free|close|...>_<extra action>`. For example: `strv_free_erase`, `iovec_done_erase`, `pidref_done_sigkill_wait`

There's also another pattern, which is `<action>_[and_<action>...]_and_<free|close|etc>`: `erase_and_free`, `rm_rf_and_free`, `unmount_and_rmdir_and_free`. Doesn't really fit in here, though IMO

Reply:
We also have `sd_bus_flush_close_unref` and friends... But the rule here is indeed pretty vague and doesn't matter too much.